lib/Doctrine/ORM/Cache/DefaultQueryCache.php

```diff
@@ -344,11 +344,9 @@ public function put(QueryCacheKey $key, ResultSetMapping $rsm, $result, array $h
      * @param array<string,mixed> $assoc
      * @param mixed               $assocValue
      *
-     * @return mixed[]|null
```

Comment:
I'd keep the annotation here. That at least makes it clear the return type wasn't forgotten here